src/opnsense/service/templates/OPNsense/Unbound/core/+TARGETS

```diff
@@ -3,6 +3,5 @@ blocklists.conf:/tmp/unbound-blocklists.conf
 dot.conf:/usr/local/etc/unbound.opnsense.d/dot.conf
 private_domains.conf:/var/unbound/private_domains.conf
 domainoverrides.conf:/usr/local/etc/unbound.opnsense.d/domainoverrides.conf
-miscellaneous.conf:/usr/local/etc/unbound.opnsense.d/miscellaneous.conf
```

Comment:
this will let the file /usr/local/etc/unbound.opnsense.d/miscellaneous.conf linger in installs.we need to ship an empty template until 23.1 is out.

Reply:
got it, thanks for the heads-up

Reply:
https://github.com/opnsense/core/pull/6053/commits/b45a4b1609cd508764f0a9adb9608181c46e1f7c